src/test/java/WinningMovieTest.java

```diff
@@ -0,0 +1,39 @@
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.SelenideElement;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static com.codeborne.selenide.Selenide.*;
+import static com.codeborne.selenide.Condition.*;
+
+public class WinningMovieTest {
+
+    @BeforeClass
+    public void setUp() {
+        Configuration.browserSize = "1920x1080";
+        Configuration.timeout = 10000; // Таймаут на 10 секунд
+        Configuration.browser = "chrome";
+    }
+
+    @Test
+    public void testSearchForTitanic() {
+
+        // Відкриття IMDb з примусовою зміною мови на англійську
+        open("https://www.imdb.com");
+        // Пошук фільму "Titanic"
+        $("input[name='q']").setValue("Titanic").pressEnter();
+
+        // Клік на фільм "Титанік" (1997)
+        $("a[href*='/title/tt0120338/']").click();
```

Comment:
That is not a very good practice for finding elements, this in a long run may change, which means test is flaky.
Instead, let's use # for id's and . for classes, example:
`$$(".ipc-metadata-list-summary-item .ipc-metadata-list-summary-item__c")
                .find(text("Titanic")).shouldHave(text("1997")).click();`
                
               
Plus, as you can see we're reusing Titanic String three times, lets instead make a  Constant, select the string and right click + refactor and select constant and add a reasonable name to it.

Reply:
When I changed , test failed and not fiding Titanic with 1997